addons/resource.language.en_gb/resources/strings.po

```diff
@@ -8951,12 +8960,12 @@ msgstr ""
 
 #: xbmc/epg/Epg.cpp
 msgctxt "#19507"
-msgid "Serious/Classical/Religious/Historical Movie/Drama"
+msgid "Serious/Classical/Religious/Historical movie/drama"
```

Comment:
whoever came up with such long EPG categories needs to be punished to death. And although It's been talked about already I also vote for spaces between the separators. I know these will become very long, but readability is horrible atm